clang/test/Analysis/Checkers/WebKit/call-args-loop-init-opaque-value.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clang_analyze_cc1 -analyzer-checker=alpha.webkit.UncountedCallArgsChecker -verify %s
```

Comment:
Do we need to test these expressions in non-trivial expressions? Or do such tests already exist?

Reply:
We have tests for non-trivial expressions: clang/test/Analysis/Checkers/WebKit/uncounted-obj-arg.cpp. I guess we can add a test case here for non-trivial case as well.

Reply:
Added a non-trivial test case.